tests/tests.rs

```diff
@@ -54,6 +54,10 @@ fn cmd_combine() {
     assert!(from_utf8(&output.stdout).unwrap().find("¡Si!").is_some());
 }
 
+static FIRST_LINE: &'static str = "[sound:empty_00060_828-00066_164.es.mp3],0:01:00.828,empty,\"<img src=\"\"empty_00063_496.jpg\"\" />\",¡Si! ¡Aang ha vuelto!,Yay! Yay! Aang\'s back!,,,¡Lo sabía!,I knew it!\n";
+
+static LAST_LINE: &'static str = "[sound:empty_00077_124-00083_379.es.mp3],0:01:17.124,empty,\"<img src=\"\"empty_00080_251.jpg\"\" />\",\"El no ha hecho nada, Sokka, fué un accidente.\",Aang didn\'t do anything.,harás que vengan a nosotros.,\"You\'re leading them straight to us, aren\'t you?\",,\n";
```

Comment:
I'd like to see some test _somewhere_ that confirms the exact format of the csv export, so we can be sure we don't break Anki support. But yeah, let's do that in a different PR in a better way than this.